keyboards/ergodox_ez/lcdsteno.c

```diff
@@ -0,0 +1,337 @@
+#include QMK_KEYBOARD_H
```

Comment:
This seems like a layout-specific hardware mod. Could we have it together with the layout?

BTW, it would be awesome to have instructions/pix in your layout readme showing how you accomplished this mod.